docs/components/page.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebarDepth: 0
+---
+
+# Page
+
+`<Page>` is a UI component that represents an application screen.
+
+---
+
+## Live Demo
+
+<DocExampleBox codeBox="https://codesandbox.io/s/n5y3lym66p?module=%2Fsrc%2FApp.vue">
```

Comment:
codebox url should be removed and you can add the live demo prop
< DocExampleBox :liveDemoMode="true" >

Reply:
line 13 has been changed